tools/license-builder.sh

```diff
@@ -100,4 +100,7 @@ addlicense "brotli" "deps/brotli" "$(cat ${rootdir}/deps/brotli/LICENSE)"
 
 addlicense "HdrHistogram" "deps/histogram" "$(cat ${rootdir}/deps/histogram/LICENSE.txt)"
 
+addlicense "node-heapdump" "src/heap_utils.cc" \
+           "$(curl -sL https://raw.githubusercontent.com/bnoordhuis/node-heapdump/master/LICENSE)"
```

Comment:
Don't you want to pin this to a commit or tag? I might yet change it to the Y1BIM (Your 1st Born Is Mine) license.

Reply:
heh... yeah, likely a good idea :-)

Reply:
```suggestion
           "$(curl -sL https://raw.githubusercontent.com/bnoordhuis/node-heapdump/0ca52441e46241ffbea56a389e2856ec01c48c97/LICENSE)"
```